src/utils/registration-helpers.ts

```diff
@@ -709,3 +710,11 @@ export const getDisabledCategories = (
 
   return disabledCategories;
 };
+
+export const findParentSubject = (json: NpiSubjectDomain[], npiSubjectHeadingId: string): string | null => {
+  const parent: NpiSubjectDomain | undefined = json.find((domain) => {
+    return domain.subdomains.some((subdomain) => subdomain.id === npiSubjectHeadingId);
+  });
+
+  return parent ? parent.id : null;
+};
```

Comment:
Typene som er angitt nå ser ut til å være de samme som de automatisk blir, så kunne vurdert å tatt de vekk
```suggestion
export const findParentSubject = (disciplines: NpiSubjectDomain[], npiSubjectHeadingId: string) => {
  const parent = disciplines.find((domain) =>
    domain.subdomains.some((subdomain) => subdomain.id === npiSubjectHeadingId)
  );
  return parent ?? null;
};
```